### ISSUE TITLE
Same script on windows and linux; having differently

### ISSUE THREAD

**Issue by rogerbirkeland** [brandon-rhodes/pyephem#115]
Hi, 
This might be related to this ticket: Getting incorrect data for near earth objects in windows(64bit) #58.

I have a script where I create one satellite from TLE, copies this object a few times and changes the ._n and ._e properties to create slightly different objects. Then, from an Observer() I calculate when the satellites are visible, and determines the period between two passes of any satellite. 

I've run this program under python3.4 and python3.5 on Ubuntu 16 and 14. I now run it on the Windows Linux Sub-system, which has python3.4 "natively", as well as python3.5 through a miniconda environment. Pyephem is installed through pip for python3.4 and conda for the miniconda environment. All versions work fine. 

On the same machine, I also run the same program on Windows, through a miniconda-environment (bundeled with Pyzo). Meaning Win 10, 64 bit. Python3.5. 

On windows, pyephem suddenly gives erronious pass/coverage calculations. I run this for a few days, generating a lot of passes. Suddenly, the coverage calculations stops, and "hangs" and always reports the same pass; it does not move further to the next pass. 

If you are interested in debugging this, I can post the full program. Should I do that?

**Comment by brandon-rhodes** [comment]
The last I checked, PyEphem did not even really work under a 64-bit Windows system — the same C code that works fine everywhere else gave bad results when compiled and run. So I am a little surprised that you have gotten pass prediction to work at all, even some of the time, since the numbers I had seen from 64-bit Windows (if I am remembering correctly?) were very far off from the numbers generated anywhere else.

The pass-calculation code is down in the `libastro` library, which I have tried to keep unchanged so that PyEphem is as compatible as possible in its results as the XEphem application whose calculations it is trying to make available in Python. So I am not sure that I am going to be allocating time towards figuring out what those C routines are doing and how they are getting hung up — I am more likely, as I continue to mature my Skyfield pure-Python astronomy library, to re-implement pass calculations there, calculations that I will own and be able to tweak and improve as much as I want!

**Comment by rogerbirkeland** [comment]
Hi, thanks for answering. Your answer is very interesting, I did not know this. I have identified when the error occurs (pass-predictions are fine up to one point), and there are something strange in the Linux-results also, at the point where the Windows results breaks. 

If you want me to, I can re-write my script a bit in order to show exactly where the event happens, and post the code here. 

If not, then I suppose the best thing is to just let it go, since you say that there are "deeper" problems with Win64. 

Again, thanks a lot for taking your time.
